### PR TITLE
Jormungandr: add shape param in places for bragi

### PR DESCRIPTION
**Pull request by TeXitoi**

### CODE REVIEW

File: source/jormungandr/tests/autocomplete_tests.py [hunk 1]
```diff
@@ -114,3 +114,19 @@ def test_place_uri_coords(self):
         places = get_not_null(response, 'places')
         assert len(places) == 1
         assert places[0]['id'] == 'stop_point:Becharles'
+
+    def test_place_shape(self):
```
Comment (antoine-de) [inline source/jormungandr/tests/autocomplete_tests.py]: can you add some comments ?

Comment (antoine-de) [reply source/jormungandr/tests/autocomplete_tests.py]: and maybe split it in multiple tests ?

File: source/navitiacommon/navitiacommon/parser_args_type.py [hunk 2]
```diff
@@ -82,26 +82,24 @@ def to_return(value):
...
+        return 'valid' in valid and (valid['valid'] == 'yes' or valid['valid'] == '')
+
+    if value:
+        if not isinstance(value, dict):
+            raise ValueError('invalid json')
+
+        if not is_geometry_valid(value) :
+            raise ValueError('invalid geojson')
+
+        geometry= value.get('geometry', {}).get('type')
```
Comment (antoine-de) [inline source/navitiacommon/navitiacommon/parser_args_type.py]: spacing

File: source/tyr/tyr/resources.py [hunk 3]
```diff
@@ -657,6 +657,9 @@ def put(self, user_id):
         if not billing_plan:
             return ({'error': 'billing_plan doesn\'t exist'}, 400)
 
+        if args['shape'] == {}:
+            args['shape'] = ujson.loads(user.shape)
```
Comment (antoine-de) [inline source/tyr/tyr/resources.py]: isn't this done by the default value ?

Comment (TeXitoi) [reply source/tyr/tyr/resources.py]: No, that's the tricky special case of the empty object saying "no modification", thus you don't have default value as you provided the empty object

**Pull request merged**